assets/new-user-experience/onboarding/steps/payment-complete/index.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * External dependencies.
+ */
+import {
+	Button,
+	Flex,
+	FlexBlock,
+	FlexItem,
+	__experimentalDivider as Divider,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import {
+	SectionTitle,
+	SectionDescription,
+} from '../../../components';
+import { RightArrowInCircle, Sync, Manage } from '../../../icons';
+import { usePaymentGatewaySettings } from '../../hooks';
+import { useSteps } from '../../hooks';
+
+export const PaymentComplete = () => {
+	const {
+		paymentGatewaySettings,
+		giftCardsGatewaySettings,
+		cashAppGatewaySettings,
+	} = usePaymentGatewaySettings();
+
+	const creditCardEnabled = paymentGatewaySettings.enabled;
+	const digitalWalletsEnabled = paymentGatewaySettings.enable_digital_wallets;
+	const giftCardsEnabled = giftCardsGatewaySettings.enabled;
+	const cashAppEnabled = cashAppGatewaySettings.enabled;
+
+	const {
+		setStep,
+	} = useSteps();
+	
+	return (
+		<>
+			<div className="woo-square-onbarding__payment-settings">
+				<div className="woo-square-onbarding__payment-settings--left">
+					<div className="woo-square-onbarding__payment-settings__intro">
+						<div className="woo-square-onbarding__payment-settings__intro--title">
+							{ __( "Congratulations,", 'woocommerce-square' ) }
+							<br />
+							{ __( "Your Payment Setup is Complete!", 'woocommerce-square' ) }
+						</div>
+						<SectionDescription>
+							{ __( "Your online store is now equipped to accept payments, making you ready to welcome customers and start generating sales.", 'woocommerce-square' ) }
```

Comment:
Seems like all these text strings can be wrapped in single quotes instead of double